src/pages/gen2/start/quickstart/index.mdx

```diff
@@ -183,38 +182,30 @@ export const data = defineData({
   },
 });
 
-/*== STEP 2 ===============================================================
-Go to your frontend source code. From your client-side code, generate a
-Data client to make CRUDL requests to your table. (THIS SNIPPET WILL ONLY
-WORK IN THE FRONTEND CODE FILE.)
+// ...
+```
 
-Using JavaScript or Next.js React Server Components, Middleware, Server
-Actions or Pages Router? Review how to generate Data clients for those use
-cases: https://docs.amplify.aws/gen2/build-a-backend/data/connect-to-API/
-=========================================================================*/
+Once you save your changes to the data model, your changes will be deployed in seconds within your cloud sandbox.
 
-/*
-"use client"
-import { generateClient } from "aws-amplify/data";
-import { type Schema } from "@/amplify/data/resource";
+Our Todo data model also has authorization set up. Our model has the `authorization` method chained to it, which you can add rules to. In our example, we are allowing an owner, or the person who creates the to do instance, to perform all actions on the data they own. We are also allowing all page viewers, including unauthenticated users,to read data. These can be modified, for example we could remove the `.to(['read'])` and allow all visitors to perform all actions on data. We could also add permissions for signed in users or users who belong to user groups such as `Admin`. You can learn more about all options for authorization in the [customize your auth rules](/build-a-backend/data/customize-authz/) section of the docs.
 
-const client = generateClient<Schema>() // use this Data client for CRUDL requests
-*/
+Let's remove public access.
 
-/*== STEP 3 ===============================================================
-Fetch records from the database and use them in your frontend component.
-(THIS SNIPPET WILL ONLY WORK IN THE FRONTEND CODE FILE.)
-=========================================================================*/
+```js
+.authorization([a.allow.owner()]),
+```
 
-/* For example, in a React component, you can use this snippet in your
-  function's RETURN statement */
-// const { data: todos } = client.models.Todo.list()
+Then, you'll see the `defineData` function, which has our schema and authorization configuration passed in as arguments. We have an `API_KEY` set up in order to enable public access. Let's change our `defaultAuthorizationMode` to `userPool` instead so that the default is to use user authentication.
```

Comment:
```suggestion
Then, you'll see the `defineData` function, which has our schema and authorization configuration passed in as arguments. We have an `apiKey` set up in order to enable public access. Let's change our `defaultAuthorizationMode` to `userPool` instead so that the default is to use user authentication.
```